### ISSUE TITLE
Future of this repo?

### ISSUE THREAD

**Issue by stevenengland** [paperless-ngx/ansible#24]
Hi guys,

do you have any plans to further support this Ansible role? I absolutely love the work you're doing with paperless-ngx and would absolutely get it if it is too much work to also support this role any further. But a statement would be great.

Anyway. As a part of a learning journey I recreated the role in my personal space. I will definitely keep on working on the role for a while because there is so much to learn :-) And because I am highly interested in a deterministic installation of paperless-ngx for LXC. So if you do not plan to fully support a Ansible role anymore I could imagine to dig into that whole thing even deeper and might maintain it for longer / broader purpose. Let me know what you think.

KR

**Comment by shamoon** [comment]
Yea, the reason this was moved to a separate repo was precisely because none of us from the core team were able / willing to maintain this. There have been a few active folks on the ansible side but sporadically. I personally don’t follow along with every PR nor is it something I can test but I’m happy to help with the practical aspects of maintenance and if anyone would like to take “point” on managing this code that would be worth discussing

**Comment by stevenengland** [comment]
Okay, thank you for the explanaination.

As I announced I put together all the stuff that I wanted to learn and combined it to setup a paperless-ngx role. It supports the complete set of configuration options of pngx 1.13.0 (at least I think so :-D) So far so good. For now I will use it to manage my own two pngx instances and will develop it in parallel to keep step with new versions of pngx. If you have any suggestions/new thoughts regarding the future of this role here please let me know. Otherwise I will be a "silent follower" to further develop my role :)

If anybody is interested: Here is the role... https://galaxy.ansible.com/stevenengland/paperless_ngx

**Comment by shamoon** [comment]
Cool. If you would like to be given an admin role in this repo to maintain it I would be totally fine with that (and I bet others would). Just let us know!

**Comment by stevenengland** [comment]
Thank you very much for the offer and the trust.  I could even imagine to go this step :-) I will think about it carefully over the weekend.

**In advance:**
Since I have no experience with maintaining a repo yet: How exactly could the scenario you describe look like? Could we write bilaterally on this?

**Questions I would have spontaneously regarding this:**
* Could someone from the original Ansible role team maybe look at/evaluate [my role](https://github.com/stevenengland/paperless_ngx_ansible_role/) to see if this is a state that is ok? If so: 
* Would it be possible to continue my codebase under the organization ` paperless-ngx`? Like by discontinuing the current repo `paperless-ngx/ansible` and pointing it to a new repo that I might maintain? For example: `paperless-ngx/paperless_ngx_ansible_role`. Why do I ask that? For one thing, this would be more verbose than the original role. But more importantly: I could clone my current repository `stevenengland/paperless_ngx_ansible_role` to `paperless-ngx/paperless_ngx_ansible_role`. Since my repo is not a direct fork of `paperless-ngx/ansible` it would be messy to try to merge both :) 

These are the ideas that came into my mind. Is there any reason not to do this? Or are there better approaches?

If all of that somehow meets your expectations, let's talk :)

**Comment by shamoon** [comment]
I think it would be better to merge your repo into this, there are simple ways to do that via git (just delete everything in one commit, then merge in yours in another). I think it would be worth it to maintain continuity.

As far as maintaining the repo its pretty much the same idea as what you're doing with your own repo. You would be given write access to this one (or higher if needed eventually) and then you can make changes, PRs etc as you see fit.

There are currently two other folks who were involved in this repo before, let's see if they chime in. If not then just let us know if you'd like to take it over. If no one is interested in doing so we should consider closing this repo at least for now.

@SiM22 @yschroeder you have both previously maintained this repository, please let us know if you're still interested in doing so and any thoughts on the above.

**Comment by SiM22** [comment]
I am all for having more hands on deck. Many hands make light work or something like that. The work I did has fallen behind the paperless-ngx releases due to nappy changing services 😝

I will take a look at the work in the repo link above and put a comment here. I am happy to help out but dedicating serious time is quite hard lately. 

I also think (if everyone agrees of course) that merging the other repo to here centralises any paperless-ngx related work rather than having seperate roles by lots of other people.

**Comment by SiM22** [comment]
The role Steven created looks a lot better than the mish-mash here. Better break down of tasks. I guess it all goes down to @stevenengland if he is happy to merge into this repo

**Comment by stevenengland** [comment]
Hi again, thank you for the warm words ;)

Well, then: Let's do it :) I agree to maintain continuity. So I would merge my repo into this one here. How can we talk about the next steps? Like getting needed resources (GH Tokens, Ansible Galaxy Token, ...)?

**Comment by shamoon** [comment]
Great! I've invited you up as a maintainer so feel free to do whatever you feel is needed to whip this repository into shape. And yes, if you need anything else you can reach out.

Thanks for your contributions!

**Comment by stevenengland** [comment]
Solved by #27
